harcownik-frontend/src/components/raport/raport_style.css

```diff
@@ -15,10 +17,11 @@
     margin-bottom: 10px;
     /* display: flex; */
     text-align: center;
+    margin-bottom: 5%;
   }
 
   .jumbotronStyle_2 {
-    height: 300px;
+    height: 400px;
```

Comment:
we was talking about using px in style files and i think is better to have device independent units.